### ISSUE TITLE
Client screen never shows on server

### ISSUE THREAD

**Issue by jbedgood** [debauchee/barrier#1149]
I've done this now multiple times with different computers (4 in total). All running windows 10. I installed barrier 2.3.3 on them and everything seems fine. The client shows it connected to the server and the server shows it sees the client. But when I try to move my mouse off screen it disappears and the client shows entering screen the server shows leaving screen but the client screen never shows on my server screen. i always just see my server screen.

Not sure what else to try at this point.

But here are some debug files.
[client-barrier.log](https://github.com/debauchee/barrier/files/6388453/client-barrier.log)
[server-barrier.log](https://github.com/debauchee/barrier/files/6388454/server-barrier.log)

**Comment by Benchwork** [comment]
yes every computer has its own screen. barrier is used to allow the keyboard and mouse to move between them instead of using multiple devices or a kvm switch.  if your running windows headless I would assume you would use something like RDP to manage them.
